logger/consoleFormatter.go

```diff
@@ -24,14 +24,30 @@ func (cf *ConsoleFormatter) Output(line LogLineHandler) []byte {
 
 	level := LogLevel(line.GetLogLevel())
 	levelColor := getLevelColor(level)
+	timestamp := displayTime(line.GetTimestamp())
+	loggerName := formatLoggerName(line.GetLoggerName())
```

Comment:
Maybe `loggerName` should be optional - it provides a lot of information, but takes up space too. If it's not too difficult now, add a configurable boolean here https://github.com/ElrondNetwork/elrond-go/blob/master/cmd/node/config/prefs.toml, called `LoggerNameInConsole` or something similar.

Once we will write the log integration tool, the `loggerName` will not be as necessary anymore.

Reply:
Made it optional.